src/client/pydaos/pydaos_core.py

```diff
@@ -281,6 +283,9 @@ class KVObj(_Obj):
     dump()
         Fetch all the key-value pairs and return them in a python dictionary.
     """
+    def __init__(self):
```

Comment:
(pylint-super-init-not-called) __init__ method from base class '_Obj' is not called